viewing_party/helper_functions.py

```diff
@@ -0,0 +1,20 @@
+"""
+This is a module that contains the helper functions implemented in the 
+viewing-party project (party.py file).
+"""
+
+from viewing_party.party import *
+from tests.test_constants import *
+
+def build_user_movie_list(user_data):
+    user_watched_movie_list = []
+    for movie in user_data["watched"]:
+        user_watched_movie_list.append(movie)
+    return user_watched_movie_list
+
+def build_friend_movie_list(user_data):
+    friends_watched_movie_list = []
+    for friend in user_data["friends"]:
+        for movie in friend["watched"]:
+            friends_watched_movie_list.append(movie)
+    return friends_watched_movie_list
```

Comment:
Pulling out helper functions into a separate file is a pattern that you'll encounter. Nice work getting practice with this. 

For a project of this scope and since there are just 2 helpers, it would be ok for these two methods to live in party.py